### ISSUE TITLE
Desperate reach not relocating from weather vane

### ISSUE THREAD

**Issue by seitaer** [PlayersCommittee/gemp-swccg-public#39]
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=engelking$0nbauaqudplrr0sx

I was playing with the weather vane and one of my arconas landed there because i played Fall of A Legend. I had Desperate Reach in hand and was unable to relocate the Arcona to a Cloud City site during my move phase on multiple tries on different turns.

First time I encountered it was on Light side turn 4:

**additional bug report**
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=loneboot$0of3ixy4lily93u7

LS turn #8 (and #9)

**Comment by Gergall** [comment]
That is correct. The code was updated on Aug 5, 2022.  Two days later, the changelog notes for Aug 7 2022 read as follows:

> 8/7/2022
> Bug Fix:
> Desperate Reach (& combo) now work properly with Weather Vane

Furthermore, I have tested and confirmed that Desperate Reach is working correctly. Closing this.
